bodhi/server/templates/update.html

```diff
@@ -523,6 +558,9 @@ <h1>
             % endif
             <a id='unpush' class="btn btn-sm btn-danger"><span class="fa fa-fw fa-arrow-circle-left"></span> Unpush</a>
           % endif
+          % if update.status.description != 'stable' and request.registry.settings.get('test_gating.required') and not update.test_gating_passed:
```

Comment:
We could make a small function in util.py that does this if block and use that over the three blocks of code in this commit.